appveyor.yml

```diff
@@ -61,26 +61,48 @@ for:
 
       # Install AWS CLI Globally outside of a venv.
       - "pip install awscli"
+      
+      # Create pre-dev virtual env without installing dev depenencies
+      # This is used for import check for prod code to prevent importing packages from dev.txt
+      - "rm -rf pre-dev-venv"
+      - "python -m virtualenv pre-dev-venv"
+      - "pre-dev-venv\\Scripts\\activate"
+      - "python --version"
+      - "deactivate"
 
-      # Create new virtual environment and activate it
+      # Create dev virtual env
       - "rm -rf venv"
       - "python -m virtualenv venv"
       - "venv\\Scripts\\activate"
       - "python --version"
+      - "deactivate"
 
     build_script:
-      # Activate virtualenv again on windows
+      # Install pre-dev dependencies
+      - "pre-dev-venv\\Scripts\\activate"
+      - "python -c \"import sys; print(sys.executable)\""
```

Comment:
nit: we could specifically say which env this is and what is the executable too for when we are debugging builds.

Reply:
Added comment on its current use.